assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -6,6 +6,7 @@ angular.module("openshiftConsole")
       $q,
       $routeParams,
       DataService,
+      ApiEndpointService,
```

Comment:
I even prefer leaving out the word 'Service' , except that we have it everywhere.  'Controller' in controllers is fine because they are never requested via DI, but we could trim these all down at some point:
```javascript
angular.module('openshiftConsole')
  .controller('foo', function (AlertMessageService, DataService, DeploymentsService, ProjectsService) {
 
// vs
angular.module('openshiftConsole')
  .controller('foo', function (Alerts, Data, Deployments, Projects) {
```